### ISSUE TITLE
VSCode doesn't recognize the library 

### ISSUE THREAD

**Issue by alexandredepaula** [robooo/robotframework-ConfluentKafkaLibrary#35]
Hello.
I have already installed both libraries:

pip install robotframework-confluentkafkalibrary==1.9.0-1
$ pip install confluent-kafka==1.9.0

But in VSCode it doesn't even know the library. And it still shows an error that seems strange to me (please, see attached image).
I already tried to install those modules that appear, but usually the error persists.

Have you ever come across something like this?

![image](https://user-images.githubusercontent.com/44452373/222788751-2db6a4e8-af69-4632-aa21-7202ac63a58d.png)

**Comment by robooo** [comment]
Hi @alexandredepaula, 
thank you for the report. I've tried to install latest vscode stable + installed library and used RF Language server by robocorp, do you use the same plugin? for me it's working.
![latest_vscode](https://user-images.githubusercontent.com/7584540/223997459-e6a0101e-f47c-467b-aacb-71cfa7779932.png)

Also I don't have anything special with python/RF in user settings.json. I have python 3.9.2 and RF4.1.3. Could you maybe try to look at your modules and maybe install bz2 missing?
https://stackoverflow.com/questions/12806122/missing-python-bz2-module

**Comment by alexandredepaula** [comment]
I reinstalled my PC (Ubuntu 22.04) and had to reinstall everything.

As much as Python 3.10 came again, I installed the Python version 3.9.16 (available), set it as default and installed RF 4.1.3 (just like you).

I am also using RF Language Server 1.9.0. I already installed the libraries: confluent-kafka "1.9.0" and robotframework-confluentkafkalibrary"1.9.0.post1".

The error still persists, but this time the message is different as below:

Unresolved library: ConfluentKafkaLibrary.
Error generating libspec:
Importing library 'ConfluentKafkaLibrary' failed: ModuleNotFoundError: No module named 'confluent_kafka.cimpl'
Consider adding the needed paths to the "robot.pythonpath" setting
and calling the "Robot Framework: Clear caches and restart" action.

Do you know if there are any other components to install or configure?

![vscode_erro](https://user-images.githubusercontent.com/44452373/225423607-653af25f-c8e9-40be-92e9-76897a5e2d11.png)

**Comment by robooo** [comment]
ok so I did more testing @alexandredepaula and it works with robot 6.0 too :wink: 
1. after uninstall of robotframework-confluentkafkalibrary I get missing module error like you have (only no bz2 issue) then I installed it again but the module not found is still there. 
2. To fix that just run from vscode command palette (ctrl+shift+p) `Robot Framework: Clear caches and restart` (as it's mentioned in error message) this will reload everything and load newly installed robot libs. :wink:
